rush.json

```diff
@@ -1,20 +1,300 @@
+/**
+ * This is the main configuration file for Rush.
+ * For full documentation, please see https://rushjs.io
+ */
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "npmVersion": "5.4.0",
+
+  /**
+   * (Required) This specifies the version of the Rush engine to be used in this repo.
```

Comment:
Can we just remove all the comments from the json file here? json files are not parse-able with comments in them, without hacks...